jetty-core/jetty-quic/jetty-quic-quiche/jetty-quic-quiche-foreign-incubator/src/main/java/org/eclipse/jetty/quic/quiche/foreign/incubator/ForeignIncubatorQuicheConnection.java

```diff
@@ -489,7 +489,7 @@ protected List<Long> iterableStreamIds(boolean write)
         try (AutoLock ignore = lock.lock())
         {
             if (quicheConn == null)
-                throw new IllegalStateException("connection was released");
+                return List.of();
```

Comment:
I'd prefer to catch from `ProtocolSession.StreamProducer.produce()`, log at debug, and return `null`.

Reply:
Ok, done.